theme/templates/includes/my-resources-trows.html

```diff
@@ -62,7 +62,7 @@
         {# Date Created #}
         <td>{{ res.created|date:"M d, Y" }} at {{ res.created|time }}</td>
         {# Last Modified #}
-        {% with updated=res.updated %}
+        {% with updated=res.last_updated %}
```

Comment:
I remember when we talked about this change...sorry to have introduced this issue!

Reply:
I think I suggested to use `updated` to avoid an additional query. I didn't realize mezzanine updates this field (`updated`), every time the resource gets saved. As one views a resource, we increment the view count for the resource and that triggers an update to the `updated `field.